update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
What I complaint here is the "inconsistency". You check for new version in one repo, and download new code from another repo!

If your work is not ready to merge (when you still use the links to your repo), don't create pull request.

And don't say that
> change the link after merging the code
the code to be merged, should be the stable one. You cannot make change after it is merged. If you found the error before it is merged, fix it right now, before it is merged (this rule is not applied for the error which is discovered after merging).